api/nodes/serializers.py

```diff
@@ -75,6 +77,9 @@ class NodeSerializer(JSONAPISerializer):
     files = JSONAPIHyperlinkedIdentityField(view_name='nodes:node-providers', lookup_field='pk', lookup_url_kwarg='node_id',
                                              link_type='related')
 
+    comments = JSONAPIHyperlinkedIdentityField(view_name='nodes:node-comments', lookup_field='pk', lookup_url_kwarg='node_id',
+                                               link_type='related', meta={'unread_comments_count': 'get_unread_comments_count'})
```

Comment:
Do we necessarily want to serialize number of unread comments every time someone requests data about a node? Would it make more sense to keep it on the linked `node-comments` page only? (per below, currently I can't find the data on either page)